internal/stream/stream.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package stream;
+
+option go_package = "github.com/windmilleng/tilt/internal/stream/proto;proto";
+
+service Stream {
+    rpc Connect (ConnectRequest) returns (stream StreamMessage) {}
+}
+
+message ConnectRequest {}
+message StreamMessage {
+    string s = 1;
```

Comment:
do we need log levels like we did last time we did this?

Reply:
I'd been figuring we'd just mirror whatever is emitted by the main tilt process, and hadn't considered honoring the -debug/etc on the stream process, but that makes more sense.